mtags/src/main/scala-2/scala/meta/internal/pc/PcCollector.scala

```diff
@@ -200,9 +200,12 @@ abstract class PcCollector[T](
       None
   }
 
-  def result(): List[T] = {
+  def result(allOccurences: Boolean = false): List[T] = {
+    if (allOccurences) resultAllOccurences().toList
+    else resultWithSought()
```

Comment:
```suggestion
    params match {
     case params: OffsetParams => resultWithSought(params.offset)
     case  _ => resultAllOccurences().toList
    }
```
we could match here instead of changing the params to OffsetParams for semantic tokens